src/shared/styles.ts

```diff
@@ -84,15 +84,15 @@ export const pageMargin = '5.55555';
 export const pageMargins = css`
   padding: 0 ${spacing.padding.medium}px;
   @media (min-width: ${breakpoint * 1}px) {
-    margin: 0 ${pageMargin * 1}%;
+    margin: 0 ${+pageMargin * 1}%;
   }
   @media (min-width: ${breakpoint * 2}px) {
-    margin: 0 ${pageMargin * 2}%;
+    margin: 0 ${+pageMargin * 2}%;
   }
   @media (min-width: ${breakpoint * 3}px) {
-    margin: 0 ${pageMargin * 3}%;
+    margin: 0 ${+pageMargin * 3}%;
   }
   @media (min-width: ${breakpoint * 4}px) {
-    margin: 0 ${pageMargin * 4}%;
+    margin: 0 ${+pageMargin * 4}%;
```

Comment:
fixes: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.